Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM fedora:27
+
+# Env variable USER specific the kebechet as committer while git branch and git commit creation. 
+ENV USER=kebechet \
+    PIPENV_CACHE_DIR=/home/user \
+    LC_ALL=en_US.UTF-8 \
+    LANG=en_US.UTF-8
+
+# Add the ssh key from local dir to container dir.
+# ADD id_rsa /home/user/.ssh/id_rsa
+
+RUN \
+    dnf install -y git python3-pip gcc redhat-rpm-config python3-devel which &&\
+    pip3 install git+https://github.com/thoth-station/kebechet &&\
```

Comment:
This is OK now - we will replace this with installing kebechet from PyPI. Let me do a release.

Reply:
Thanks, i'll make the changes.